src/routes/search/containers/SearchPage.js

```diff
@@ -43,10 +45,13 @@ export class SearchPage extends React.Component<PropsType, LocalStateType> {
       .sort((category1, category2) => category1.title.localeCompare(category2.title))
 
     // find all categories whose contents but not titles include the filter text and sort them lexicographically
-    const categoriesWithContent = categories.toArray()
-      .filter(category => !normalizeSearchString(category.title).includes(filterText))
-      .filter(category => normalizeSearchString(category.content).includes(filterText))
-      .sort((category1, category2) => category1.title.localeCompare(category2.title))
+    // only filter content if the filter text is at least 3 characters long due to performance issues
+    const categoriesWithContent = filterText.length < CONTENT_MIN_FILTER_LENGTH ? []
+      : categories.toArray()
+        .filter(category => !normalizeSearchString(category.title).includes(filterText))
+        .filter(category => normalizeSearchString(htmlToText(category.content,
```

Comment:
So `htmlToText` is ran twice for a search, right? Would it be possible to change this so that is only ran once?

Reply:
Also, only filtering for the content starting after three characters may seem weird for users. I think we should search for a better solution.